src/scripts/build_all_repos.sh

```diff
@@ -908,6 +911,23 @@ EOF
 
 }
 
+function test_rpms() {
+
+    # Not the fastest way ever
+    for file in $NOREPLACE_FILES; do
+        for rpm in `ls $RPMS/*rpm`; do
```

Comment:
the overkill is negligible. it takes 4.6 seconds to run the test on all 15.3 rc3 rpms on a 1 core VM